### DISCUSSION TITLE
I keep getting prompted to "Pick how to connect to Jupyter" even though no .ipynb files are open

### DISCUSSION THREAD

**Discussion by thatlittleboy** [microsoft/vscode-jupyter#9920] [Questions and Answers] [answered]
<img width=400 src="https://user-images.githubusercontent.com/30731072/167106432-773346ce-4824-4ec9-9a02-a58e87a83eaa.png">

I've just updated VS Code to the latest version, and also the Jupyter extension.

And now everytime I open up a vscode window with a python file, I will get prompted to "Pick how to connect to Jupyter", when I have no intention to use Jupyter whatsoever.
It's not clear from my image, but I assure you I have no ipynb files open, nor anywhere in my Code workspace.

How can I get the extension to stop prompting me everytime? OR at least restrict it to prompt me only when a Jupyter server is actually required, i.e., when I open an ipynb file?

Thanks!

## Version info

**VS-code**
Version: 1.67.0
Commit: 57fd6d0195bb9b9d1b49f6da5db789060795de47
Date: 2022-05-04T12:14:47.100Z (1 day ago)
Electron: 17.4.1
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.13-electron.0
OS: Darwin x64 21.3.0

**Jupyter extension** v2022.4.1001271738

**Comment by rchiodo** [comment, marked as answer]
Sorry this is actually a bug being tracked here:
#9167 

The way to get that dialog to disappear is to run 'Jupyter: Specify jupyter server for connections' and pick none. 

@DonJayamanne's change will make it possible to only have the connection failure show up when running (instead of randomly)
